astropy/io/fits/tests/test_image.py

```diff
@@ -868,7 +868,6 @@ def test_rewriting_large_scaled_image(self):
         hdul.close()
         hdul = fits.open(self.temp('test_new.fits'))
         assert (hdul[0].data == orig_data).all()
-        hdul = fits.open(self.temp('test_new.fits'))
```

Comment:
Are you sure the second `open` wasn't intentional to test something? I never know with FITS.

Reply:
I have no idea why the file was opened again, but it seemed unintentional (what would it test).  It was intentional, then the file will need to be closed.

Reply:
@embray , do you recall this? 😸

Reply:
Looks like a copy/paste error.